crowdnotifier-sdk/src/main/proto/backend_v3.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
+package org.crowdnotifier.android.sdk.model;
+
+message QRCodeTrace {
```

Comment:
Shouldn't all these protobufs be of type `*V3`? If the project continues, I'm sure we'll come up with `V4`. Or did I miss some discussion here?

Reply:
naming is to align with others. But package versioning could work

Reply:
Yes I agree! Let's add a V3 suffix to all Protobufs used in V3. @wouterl @martinalig @ubfelix Will you do this in your Code as well?

Reply:
I would _not_ rename the protobuf